test/valid-data/type-partial/main.ts

```diff
@@ -0,0 +1,4 @@
+export interface MyObject extends Partial<{
+    numberField: number,
+    stringField: string
```

Comment:
Does this work if there is only one property?

Reply:
I'll have to check.